eval/adjust.rkt

```diff
@@ -138,6 +138,7 @@
   (define hi (ival-hi x))
   (cond
     ; x = [0.bf, ...]
+    [(and (bfzero? lo) (bfzero? hi)) (get-slack)]
```

Comment:
Hmm, can it also be that we have an interval `[inf, inf]` or `[-inf, -inf]`?
I know that it is commented below as `not a valid interval`, but anyways

Reply:
I think no, or at least it should never happen? Because logically that interval contains no real numbers (`inf` is not a real number) so it's an empty interval.

Reply:
I think that's true, that never happens